test/wpt/README.md

```diff
@@ -148,32 +154,37 @@ expected failures.
 
 <a id="status-format"></a>
 
-## Format of a status JSON file
+## Format of a status file
 
-```text
-{
-  "something.scope.js": {  // the file name
+<!-- eslint-skip -->
+
+```js
+module.exports = {
+  'something.scope.js': { // the file name
     // Optional: If the requirement is not met, this test will be skipped
-    "requires": ["small-icu"],  // supports: "small-icu", "full-icu"
+    requires: ['small-icu'], // supports: 'small-icu', 'full-icu'
 
     // Optional: the test will be skipped with the reason printed
-    "skip": "explain why we cannot run a test that's supposed to pass",
+    skip: "explaination why we cannot run a test that's supposed to pass",
 
     // Optional: failing tests
-    "fail": {
-      "note": "You may leave an optional arbitrary note e.g. with TODOs",
-      "expected": [
-        "test name in the WPT test case, e.g. second argument passed to test()",
-        "another test name"
+    fail: {
+      expected: [
+        'test name in the WPT test case, e.g. second argument passed to test()',
+        'another test name',
       ],
-      "flaky": [
-        "flaky test name"
-      ]
-    }
-  }
-}
+      flaky: [
+        'flaky test name',
+      ],
+    },
+  },
+};
 ```
 
+You may automatically format the files by running `make lint-js-fix`. The lint
```

Comment:
It seems a bit weird to mention this command here, as this is not specific to WPT and applies to every js file in the code base..